app/code/core/Mage/Adminhtml/Block/System/Config/Edit.php

```diff
@@ -65,7 +65,22 @@ protected function _prepareLayout()
      */
     public function getSaveButtonHtml()
```

Comment:
I think it's better to create a separate method, something like `public function getAdditionalButtons()`, and call it in `app\design\adminhtml\default\default\template\system\config\edit.phtml`.

Reply:
done